docs/sources/panels-visualizations/visualizations/state-timeline/index.md

```diff
@@ -19,9 +19,55 @@ weight: 100
+
+{{< figure src="/static/img/docs/state-timeline-panel/state-timeline-panel.png" max-width="1025px" alt="A state timeline panel showing CPU usage" >}}
+
+The state timeline visualization is useful when you need to monitor and analyze changes in states or statuses of various entities over time. You can use one when you need to:
+
+- Monitor the status of a server, application, or service to know when your infrastructure is experiencing issues over time.
+- Identify operational trends over time.
+- Spot any recurring issues with the health of your applications.
+
+## Configure a state timeline
+
+<!-- video TBA here -->
+
+## Supported data formats
+
+The state timeline panel works best if you have data capturing the various states of entities over time, formatted as a table. The data must include:
+
+- **Timestamps** - Indicate when each state change occurred. This could also be the start time for the state change. You can also add an optional timestamp to indicate the end time for the state change.
+- **Entity name/identifier** - Represents the name of the entity you're trying to monitor.
+- **State value** - Represents the state value of the entity you're monitoring. These can be string, numerical, or boolean states.
+
+Each state ends when the next state begins or when there is a `null` value.
+
+### Example
+
+The following table is an example of the type of data you need for a state timeline visualization and how it should be formatted:
+
+| Timestamps          | Server A | Server B |
+| ------------------- | -------- | -------- |
+| 2024-02-29 8:00:00  | Up       | Up       |
+| 2024-02-29 8:15:00  | null     | Up       |
+| 2024-02-29 8:30:00  | Down     | null     |
+| 2024-02-29 8:45:00  |          | Up       |
+| 2024-02-29 9:00:00  | Up       |          |
+| 2024-02-29 9:15:00  | Up       | Down     |
+| 2024-02-29 9:30:00  | Up       | Down     |
+| 2024-02-29 10:00:00 | Down     | Down     |
+| 2024-02-29 10:30:00 | Warning  | Down     |
+
+The data is converted as follows, with the [null and empty values visualized as gaps](https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/state-timeline/#connect-null-values) in the state timeline:
+
+{{< figure src="/static/img/docs/state-timeline-panel/state-timeline-with-null-values.png" max-width="1025px" alt="A state timeline panel with null values showing the status of two servers" >}}
+
+{{< admonition type="note" >}}
+If your query results aren't in a table format like the preceding example, especially for time-series data, you can apply [transformations](https://grafana.com/docs/grafana/latest/panels-visualizations/query-transform-data/transform-data/) to achieve this.
+{{< /admonition >}}
```

Comment:
I'm not sure this needs to be a note. I think it's fine for it to be part of the regular body text since it's integral to getting the right type of data for this visualization type. 

I moved this note down so that it doesn't interrupt the flow of seeing the table data visualized in the image. 

```suggestion
If your query results aren't in a table format like the preceding example, especially for time-series data, you can apply [transformations](https://grafana.com/docs/grafana/latest/panels-visualizations/query-transform-data/transform-data/) to achieve this.
```